package.json

```diff
@@ -169,6 +169,7 @@
     "eslint": "^6.8.0",
     "eslint-config-google": "^0.14.0",
     "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-import": "^2.22.0",
```

Comment:
If this is being added for the functions' template, doesn't this need to go in `templates/init/functions/typescript/package.lint.json`?

Reply:
+1 this - not sure if this needs to be included in the top level package.json.

Reply:
`npm run lint:changed-files` needs this package even if this is being added for functions' template. CI failed due to the lack of this package. https://github.com/firebase/firebase-tools/runs/944773354

Reply:
Ah, but in the logs, it says:

> The plugin "eslint-plugin-import" was referenced from the config file in "templates/init/functions/typescript/.eslintrc.js".

What I think is happening is that that `.eslintrc.js` file is being used by `eslint` in the lint script and changing that rule. `firebase-tools` still does not need this plugin.

What I would recommend is changing the eslint template's filename to be `_eslintrc.js` (or `_eslintrc_js` if it still breaks the linter) so that the repo's `eslint` does not pick it up and try to use it.

If _that_ doesn't work, go into `lint-changed-files.ts` and add a smidge of logic to ignore the `templates` directory - we already ignore them in `.eslintignore`, so I think that would be reasonable to do (it's not being ignored because `lint-changed-files` invokes `eslint` with exact file names, ignoring `.eslintignore`)

Sorry you ran into this corner case! Thanks for taking the time to work through this with us.